src/bailandopp/app/requirements.txt

```diff
@@ -0,0 +1,72 @@
+aiofiles==23.2.1
```

Comment:
let's do pipfile. Need to look into how to have 2 pipenvs in the same place. One will be for cuda and one for nightly.